src/uni_modules/wot-design-uni/components/common/props.ts

```
@@ -39,7 +39,18 @@ export const makeStringProp = <T>(defaultVal: T) => ({
   default: defaultVal
 })
 
-export const baseProps = {
+export type IBaseProp = {
+  customStyle: {
+    type: PropType<import('@vue/runtime-dom/dist/runtime-dom.d.ts').StyleValue>
+    default: string
+  },
```

ESLint

[error] Delete `,`

(prettier/prettier)